crates/cast/Cargo.toml

```diff
@@ -16,7 +16,7 @@ url.workspace = true
 rand.workspace = true
 project-root.workspace = true
 which.workspace = true
-scarb-metadata = "=1.4.2"
+scarb-metadata = { git = "https://github.com/software-mansion/scarb.git", rev = "58cc88e" }
```

Comment:
I'd prefer this included changes suggested in https://github.com/foundry-rs/starknet-foundry/issues/675 instead

Reply:
Why do we need to set it to commit instead of version?

Reply:
@MaksymilianDemitraszek we don't need to anymore, we needed it before with Scarb 0.7.0 bcs scarb-ui was not published on crates.io and is dependent on metadata. As @THenry14 says, #675 is related to that

Reply:
Done in #1019